container/container-core/src/main/java/org/talend/sdk/component/container/ContainerManager.java

```diff
@@ -110,14 +110,11 @@ public ContainerManager(final DependenciesResolutionConfiguration dependenciesRe
         this.logInfoLevelMapping = logInfoLevelMapping;
         this.containerInitializer = containerInitializer;
         this.resolver = dependenciesResolutionConfiguration.getResolver();
-        this.rootRepositoryLocation = ofNullable(dependenciesResolutionConfiguration.getRootRepositoryLocation())
-                .filter(Files::exists)
-                .orElseGet(() -> PathFactory.get(System.getProperty("user.home", "")).resolve(".m2/repository"));
-
-        if (log.isDebugEnabled()) {
-            log.debug("Using root repository: " + this.rootRepositoryLocation.toAbsolutePath());
-        }
+        this.rootRepositoryLocation = dependenciesResolutionConfiguration.getRootRepositoryLocation();
 
+        info("Using root repository: " + this.rootRepositoryLocation.toAbsolutePath());
+        final String pluginsLocation = System.getProperty("talend.component.manager.plugins.location",
```

Comment:
Sonar reports that this system property isn't used.
I guess it might not be expected. 
How do we plan to use it?